PLAYER/GetPlayersLastVehicle.md

```diff
@@ -8,8 +8,11 @@ ns: PLAYER
 Vehicle GET_PLAYERS_LAST_VEHICLE();
 ```
 
-```
-Alternative: GET_VEHICLE_PED_IS_IN(PLAYER_PED_ID(), 1);  
-```
+Will return the player's last vehicle. The native will return `0` if the last vehicle the player was in was destroyed.
+
+This native accesses a global variable which gets unset by the game when a vehicle is destroyed (also gets unset in many other places), so it's not useful for cases where vehicles are destroyed.
```

Comment:
Redundant information

Reply:
Will remove the redundancy, but will still warn that users shouldn't use this native if they plan to access a 'last vehicle' that was destroyed by who knows 😀